components/clearbit/actions/find-contacts/find-contacts.mjs

```diff
@@ -0,0 +1,112 @@
+import app from "../../clearbit.app.mjs";
+import options from "../../common/options.mjs";
+
+export default {
+  key: "clearbit-find-contacts",
+  name: "Find Contacts",
+  description: "Find people who work at a specific company. [See the docs here](https://dashboard.clearbit.com/docs#prospector-api)",
+  version: "0.2.0",
```

Comment:
Is using `version: "0.2.0"` ok here? Since key `clearbit-find-contact` changed to `clearbit-find-contacts`, would it be better to have something like `0.0.1`?